src/scales/scale.time.js

```diff
@@ -704,13 +704,14 @@ module.exports = Scale.extend({
 	 */
 	getPixelForOffset: function(time) {
 		var me = this;
-		var isReverse = me.options.ticks.reverse;
+		var offsets = me._offsets;
 		var size = me._horizontal ? me.width : me.height;
-		var start = me._horizontal ? isReverse ? me.right : me.left : isReverse ? me.bottom : me.top;
 		var pos = interpolate(me._table, 'time', time, 'pos');
-		var offset = size * (me._offsets.start + pos) / (me._offsets.start + 1 + me._offsets.end);
+		var offset = size * (offsets.start + pos) / (offsets.start + 1 + offsets.end);
```

Comment:
Everytime `offsets.end` is used we do `(offsets.start + 1 + offsets.end)`. Perhaps we should just store that as `offsets.offsetSize` or something instead so that we don't have to recompute?

Reply:
Agreed. `offsets.factor` is added to store `1 / (offsets.start + 1 + offsets.end)`.